CONTRIBUTING.rst

```diff
@@ -0,0 +1,241 @@
+  <http://stackoverflow.com/questions/23019608/github-commit-syntax-to-link-a-pull-request-issue>`__.
+- Keep your pull requests as small as possible. Large pull requests
+  are hard to review. Try to break up your changes into self-contained
+  and incremental pull requests.
+- The first line of commit messages should be a short (<80 character)
+  summary, followed by an empty line and then any details that you
+  want to share about the commit.
+- Please try to follow the `existing syntax style <#syntax_style>`__
+
+When you submit or change your pull request, the Travis build system
+will automatically run tests to ensure valid schema syntax. If your
+pull request fails to pass tests, review the test log, make changes
+and then push them to your feature branch to be tested again.
+
+Issue Resolution
+@@@@@@@@@@@@@@@@
+
+Once a pull request or issue have been submitted, anyone can comment or
+vote on to express their opinion following the Apache voting system.
+Quick summary:
+
+- **+1** something you agree with
+- **-1** if you have a strong objection to an issue, which will be
+  taken very seriously. A -1 vote should provide an alternative
+  solution.
+- **+0** or **-0** for neutral comments or weak opinions.
+- It's okay to have input without voting
+- Silence gives assent
+
+A pull request with at least two **+1** votes, no **-1** votes, that has
+been open for at least 3 days, and whose cross-referenced pull requests
+to server and compliance have similarly been upvoted is ready to be
+merged. The merge should be done by someone from a different
+organization than the submitter. (We sometimes waive the 3 days for
+cosmetic-only changes -- use good judgment.) A pull request to either
+the schemas, servers or compliance repository that involves changes to
+the others should not be merged without coordinating, mergable pull
+requests to the other repositories. Conversely, when merging a pull
+request the other pull requests in the branch set must be merged at the
+same time (In practise, when merging a branch set including the schemas
+repository, merge the pull request to schemas first to avoid the
+continuous integration build issues). If an issue gets any **-1** votes,
+the comments on the issue need to reach consensus before the issue can
+be resolved one way or the other. There isn't any strict time limit on a
+contentious issue.
+
+The project will strive for full consensus on everything until it runs
+into a problem with this model.
+
+Syntax Style and Conventions
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+The current code conventions for the source files are as follows:
+
+- Use two-space indentation, and no tabs.
+- Hard-wrap code to 80 characters per line.
+- Use ``UpperCamelCase`` for object or record names.
+- Use ``lowerCamelCase`` for attribute or method names.
+- Use ``CONSTANT_CASE`` for global and constant values.
+- Comments:
+
+- Comments should be indented at the same level as the surrounding
```

Comment:
In Markdown, this was a sub-bullet of `Comments:`. In RST this has become a normal bullet:
https://github.com/reece/schemas/blob/documentation/CONTRIBUTING.rst#syntax-style-and-conventions